src/Core/Checker.cpp

```diff
@@ -21,10 +21,10 @@
 #include "Core/MessageLogger.hpp"
 #include "Core/Runner.hpp"
 #include "Util/FileUtil.hpp"
+#include "third_party/testlib/testlib.h"
```

Comment:
Full name to library makes sure that clang-tidy knows this header is third_party and it should not check for issues from this header. It is due to this that all third_party headers now have to start from `third_party/dep/header.hpp` instead of just `header.hpp`.